src/lib/block/Calendar/feedLoader/module.js

```diff
@@ -0,0 +1,98 @@
+import { makeColors } from './colors'
+import Record from 'corteza-webapp-common/src/lib/types/compose/record'
+import { removeDup } from 'corteza-webapp-compose/src/common/scripts'
+import variables from 'corteza-webapp-compose/src/themes/corteza-base/variables.scss'
+
+const defaultColor = variables.primary
+
+/**
+ * Loads & converts module resource into FC events
+ * @param {ComposeAPI} $ComposeAPI ComposeAPI provider
+ * @param {Module} module Current module
+ * @param {Namespace} namespace Current namespace
+ * @param {Feed} feed Current feed
+ * @param {Object} range Current date range
+ * @returns {Array} A set of FC events to display
```

Comment:
returns a promise?

Reply:
Aha! Was merely testing if you'd spot the strategically placed mistake